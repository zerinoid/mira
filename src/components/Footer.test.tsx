import { render, screen } from '@testing-library/react'
import { MemoryRouter as Router } from 'react-router-dom'
import Footer from './Footer'

describe('Test Footer comp', () => {
  it('Should render footer comp', () => {
    const { container } = render(
      <Router>
        <Footer />
      </Router>
    )
    const footerText = screen.getByText(/zerino desenvolvimento/i)
    expect(container).toMatchSnapshot()
    expect(footerText).toBeTruthy()
  })
})
