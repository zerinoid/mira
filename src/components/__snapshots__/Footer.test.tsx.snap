// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test Footer comp > Should login 1`] = `
<div>
  <footer
    class="font-sans"
  >
    <div
      class="text-sm text-bold container flex items-center justify-center h-12 px-[5px] md:justify-end"
    >
      <div>
        <a
          href="https://zerino.org"
          rel="noreferrer"
          target="_blank"
        >
          zerino desenvolvimento
        </a>
      </div>
    </div>
  </footer>
</div>
`;

exports[`Test Footer comp > Should render footer comp 1`] = `
<div>
  <footer
    class="font-sans"
  >
    <div
      class="text-sm text-bold container flex items-center justify-center h-12 px-[5px] md:justify-end"
    >
      <div>
        <a
          href="https://zerino.org"
          rel="noreferrer"
          target="_blank"
        >
          zerino desenvolvimento
        </a>
      </div>
    </div>
  </footer>
</div>
`;
